airflow/providers/common/sql/hooks/sql.py

```diff
@@ -515,3 +516,55 @@ def test_connection(self):
             message = str(e)
 
         return status, message
+
+    def get_database_info_for_openlineage(self, connection):
+        """
+        Should return database specific information needed
+        to generate and parse lineage metadata.
+        This includes information helpful for constructing information schema query
+        and creating correct namespace.
+
+        :param connection: Airflow connection to reduce calls of `get_connection` method
+        :return: A :class:`airflow.providers.openlineage.sqlparser.DatabaseInfo` instance.
+        """
+
+    def get_database_dialect(self, connection) -> str:
```

Comment:
What do you think if suffixing all open lineage-specific methods with `openlineage`?

Reply:
I like putting `openlineage` after `get_`; reads better than a suffix.

Reply:
Done.